### ISSUE TITLE
Initial plotting test issues

### ISSUE THREAD

**Issue by kbhargava** [travissluka/socaplot#2]
1. In reference to [this line](https://github.com/travissluka/socaplot/blob/00e3ca2cb36b0ddaf7671be5716675e09fa61550/src/socaplot/bin/plot.py#L70): we don't have a variable named `$EXP_NAME` in our `exp.config `

**Comment by travissluka** [comment]
@kbhargava yes, you'll have to add it. just pick something short but unique

**Comment by travissluka** [comment]
https://github.com/travissluka/socaplot/blob/00e3ca2cb36b0ddaf7671be5716675e09fa61550/src/socaplot/bin/plot.py#L32
